vector/src/main/java/im/vector/app/features/roommemberprofile/RoomMemberProfileViewState.kt

```diff
@@ -33,6 +33,7 @@ data class RoomMemberProfileViewState(
         val isMine: Boolean = false,
         val isIgnored: Async<Boolean> = Uninitialized,
         val isRoomEncrypted: Boolean = false,
+        val isAlgorithmSupported: Boolean = true,
```

Comment:
Will not change anything, but maybe set to `false` by default?

Reply:
Not sure, in the nominal case it will be true

Reply:
OK